tests/web/rpc/v1/test_endpoint_trace_item_table/test_endpoint_trace_item_table.py

```diff
@@ -2328,6 +2328,203 @@ def test_bad_aggregation_filter(self, setup_teardown: Any) -> None:
         with pytest.raises(BadSnubaRPCRequestException):
             EndpointTraceItemTable().execute(message)
 
+    def test_aggregation_filter_with_binary_formula(self) -> None:
```

Comment:
Should we add a test where `aggregation` and `formula` are both set to assert that it raises exception?